clustered/client/src/main/java/org/ehcache/clustered/client/config/Timeouts.java

```diff
@@ -0,0 +1,150 @@
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.ehcache.clustered.client.config;
+
+import org.ehcache.clustered.client.internal.ClusterTierManagerClientEntity;
+
+import java.time.Duration;
+import java.util.Objects;
+
+import static java.time.temporal.ChronoUnit.SECONDS;
+
+/**
+ * Describes the timeouts for {@link ClusterTierManagerClientEntity} operations.  Use
+ * {@link #builder()} to construct an instance.
+ */
+public final class Timeouts {
+
+  public static final Duration DEFAULT_OPERATION_TIMEOUT = Duration.of(5, SECONDS);
+
+  private final Duration readOperationTimeout;
+  private final Duration mutativeOperationTimeout;
+  private final Duration lifecycleOperationTimeout;
+
+  private Timeouts(Duration readOperationTimeout, Duration mutativeOperationTimeout, Duration lifecycleOperationTimeout) {
+    this.readOperationTimeout = readOperationTimeout;
+    this.mutativeOperationTimeout = mutativeOperationTimeout;
+    this.lifecycleOperationTimeout = lifecycleOperationTimeout;
+  }
+
+  public Duration getReadOperationTimeout() {
+    return readOperationTimeout;
+  }
+
+  public Duration getMutativeOperationTimeout() {
+    return mutativeOperationTimeout;
+  }
+
+  public Duration getLifecycleOperationTimeout() {
+    return lifecycleOperationTimeout;
+  }
+
+  @Override
+  public boolean equals(Object o) {
+    if (this == o) return true;
+    if (o == null || getClass() != o.getClass()) return false;
+
+    Timeouts timeouts = (Timeouts) o;
+
+    if (!readOperationTimeout.equals(timeouts.readOperationTimeout)) {
+      return false;
+    }
+    if (!mutativeOperationTimeout.equals(timeouts.mutativeOperationTimeout)) {
+      return false;
+    }
+    return lifecycleOperationTimeout.equals(timeouts.lifecycleOperationTimeout);
+  }
+
+  @Override
+  public int hashCode() {
+    int result = readOperationTimeout.hashCode();
+    result = 31 * result + mutativeOperationTimeout.hashCode();
+    result = 31 * result + lifecycleOperationTimeout.hashCode();
+    return result;
+  }
+
+  public static Builder builder() {
+    return new Builder();
+  }
+
+  @Override
+  public String toString() {
+    return "Timeouts{" +
+        "readOperationTimeout=" + readOperationTimeout +
+        ", mutativeOperationTimeout=" + mutativeOperationTimeout +
+        ", lifecycleOperationTimeout=" + lifecycleOperationTimeout +
+        '}';
+  }
+
+  /**
+   * Constructs instances of {@link Timeouts}.  When obtained from
+   * {@link Timeouts#builder()}, the default values are pre-set.
+   */
+  public static final class Builder {
```

Comment:
This could implement  `org.ehcache.config.Builder<Timeouts>`

Reply:
ok